tests/foreman/cli/test_realm.py

```diff
@@ -290,5 +285,5 @@ def test_negative_info_nonexistent_realm_id(self):
 
         :expectedresults: Realm not found
         """
-        with self.assertRaises(CLIReturnCodeError):
+        with pytest.raises(CLIReturnCodeError):
             Realm.info({'id': 0})
```

Comment:
Good Candidate, to merge the test case?

Reply:
I see what you mean but I suppose there is not much to gain here time-wise